packages/client/lib/net/protocol/ethprotocol.ts

```diff
@@ -6,7 +6,8 @@ import {
   BlockBodyBuffer,
 } from '@ethereumjs/block'
 import { TransactionFactory, TypedTransaction } from '@ethereumjs/tx'
-import { bigIntToBuffer, bufferToBigInt, bufferToInt, intToBuffer, rlp } from 'ethereumjs-util'
+import { encodeReceipt } from '@ethereumjs/vm/dist/runBlock'
```

Comment:
Is there an easy way to move `encodeReceipt` to a package level export since we're using this outside of the `vm` package and this import looks kinda hideous, maybe in a new `util.ts` since this looks like a pure function?  This could be a follow-up as part of a more general overhaul of the vm package getting away from default exports and definitely shouldn't hold up merging this.

Reply:
yeah, good eye, this isn't as scary as it looks (we access vm/dist in a few other places) but definitely a good point to ease access a bit, i think this is likely what holger was aiming with this breaking releases task:

> - [ ] Restructure/sort VM exports (directly expose Bloom, other?, unified type location, type/interface exposure, index.ts -> vm.ts+ index.ts only for exports